docs/reference/builder.md

```diff
@@ -123,8 +128,97 @@ be treated as an argument. This allows statements like:
     # Comment
     RUN echo 'we are running some # of cool things'
 
-Here is the set of instructions you can use in a `Dockerfile` for building
-images.
+## Parser directives
+
+Parser directives affect the way in which subsequent lines in a `Dockerfile`
+are handled. Parser directives do not add layers to the overall build, and
+will not be shown as a step in the builder. Parser directives must appear 
+before any builder instruction in a Dockerfile, and are distinguised from
+commands by having the `%` prefix.  The following parser directive is supported:
```

Comment:
Perhaps add a line to say that line-continuation chars are not supported in parser directives - so:
```
%ESCAPE \
`
```
is invalid.